drivers/usb/udc/udc_mcux_ehci.c

```diff
@@ -525,10 +574,10 @@ usb_status_t USB_DeviceNotificationTrigger(void *handle, void *msg)
 	case kUSB_DeviceNotifyLPMSleep:
 		break;
 	case kUSB_DeviceNotifyDetach:
-		udc_submit_event(dev, UDC_EVT_VBUS_REMOVED, 0);
+		udc_mcux_change_state(dev, priv->enabled, 0);
 		break;
 	case kUSB_DeviceNotifyAttach:
-		udc_submit_event(dev, UDC_EVT_VBUS_READY, 0);
+		udc_mcux_change_state(dev, priv->enabled, 1);
```

Comment:
same as above

Reply:
In case github gets busy, the change in question is being addressed on line 109